internal/networkmap/did.go

```diff
@@ -89,6 +91,15 @@ func (nm *networkMap) generateDIDAuthentication(ctx context.Context, identity *c
 	}
 }
 
+func (nm *networkMap) generateCardanoAddressVerifier(identity *core.Identity, verifier *core.Verifier) *VerificationMethod {
+	return &VerificationMethod{
+		ID:                  verifier.Hash.String(),
+		Type:                "PaymentVerificationKeyShelley_ed25519", // hope that it's safe to assume we always use Shelley
```

Comment:
What happenes when it's a script witness?

Reply:
Can that happen? I thought DID documents were for identities attached to firefly nodes, i.e. the wallets they could spend from.

Reply:
Yeah that is correct